core/store/presenters/presenters.go

```diff
@@ -334,9 +336,11 @@ func (i Initiator) MarshalJSON() ([]byte, error) {
 	}
 
 	return json.Marshal(&struct {
+		ID     int64       `json:"id"`
```

Comment:
It wasn't clear to me why we left this out before - seems confusing to return an empty ID/jobID in some parts of the JSON response?